docs/en/operation/Scalability-Tuning.md

```diff
@@ -29,12 +29,12 @@ A third party metrics collector can be used to monitor the rate of
 change of this metric to determine how the number of files are growing over time.
 
 The number of files in Alluxio impacts the following:
-* Size of heap required by the master - Each file takes approximately 1 - 2 KB. If RocksDB is used,
-most file metadata is stored off-heap, and the size of the heap impacts how many files’ metadata can
-be cached on heap. See the
+* Size of heap required by the master - Each file and its directory structure takes approximately 4KB. 
```

Comment:
Do you mean the file plus what this file will incur on the directory equals 4KB? I was first told the size is 1KB 2 years back, then I saw this 1-2KB last year, then I see this 4KB now... I'm a little confused based on the current doc. And probably that implies this  part should be further elaborated. What do you think?

Reply:
the inode itself is still estimated to take 2KB per inode, but we need to consider all the other overhead, like the directory structure, and JVM heap usage. we used 4KB in our latest calculation for Rocksdb cache, that's why i adjusted to 4kb here for consistency.